themes/jb/layouts/partials/episode/tags.html

```diff
@@ -1,6 +1,6 @@
 {{ range $tag := . }}
   <span class="tag">
     <!-- urlquery found here: https://discourse.gohugo.io/t/url-encoding-percent-encoding-with-hugo/16546/13 -->
-    <a href="{{ `/tags/` }}{{ urlquery $tag }}/">{{$tag}}</a>
+    <a href="{{ `/tags/` }}{{ replace $tag " " "-" | urlquery }}/">{{$tag}}</a>
```

Comment:
Is there a reason why you used `urlquery` instead of `urlize`? This function seems to replace hyphens with spaces internally, so the syntax is cleaner to read.
https://gohugo.io/functions/urlize/

Reply:
`urlquery` won in favor of `urlize` in #319